### DISCUSSION TITLE
SSH,SNMP and PING other network T-POT

### DISCUSSION THREAD

**Discussion by kalogirev2** [telekom-security/tpotce#1378] [Q&A]
![network](https://github.com/telekom-security/tpotce/assets/93402399/f33f624d-2106-4421-9a46-e5a4a7ee4690)

My T-POT machine is currently in a DMZ with IP 172.16.0.1 and I have a supervision machine in my personal network with IP 192.168.1.117. Between the two there's also a firewall (stormshield). From the 172.16.0.0/24 network I can do what I want on the T-POT machine, but from the 192.168.1.0/24 network I can't access it despite good firewall rules.

Can you tell me what to do on the T-POT machine to authorize the machine on the 192.168.1.0/24 network to access it for pinging, SNMP and SSH:?

Thanks in advance

Best regards

**Comment by Ganesh-403** [comment]
@kalogirev2 
What you're describing is a common deployment scenario: **T-Pot in a DMZ**, with a separate internal management workstation.

The good news is that this is supported, but there are a couple of things to keep in mind.

### 1. Verify you're connecting to the correct management ports

By default, T-Pot does **not** expose standard SSH (`22`) or its Web UI on the usual ports. Instead it uses high-numbered management ports, for example:

- SSH: **64295**
- Web UI: **64297**

The official documentation also recommends placing T-Pot behind a firewall and allowing access to these management ports **only from trusted IPs**, while forwarding the honeypot ports separately. :contentReference[oaicite:0]{index=0}

For SSH, for example:

```bash
ssh -p 64295 <user>@172.16.0.1
```

---

### 2. Check T-Pot's firewall rules

T-Pot manages its own firewall rules, so even if your Stormshield firewall allows the traffic, the host itself may still block it.

Check which firewall backend is active:

```bash
sudo nft list ruleset
```

or, on older installations:

```bash
sudo iptables -L -n -v
```

If needed, you can add a rule allowing only your management subnet or workstation to access the required management ports. For example, with nftables:

```nft
ip saddr 192.168.1.117 tcp dport { 64295, 64297 } accept
```

This is generally preferable to opening access for the entire network. The nftables documentation has examples of matching traffic by source IP and destination port. :contentReference[oaicite:1]{index=1}

---

### 3. About Ping and SNMP

- **Ping (ICMP):** If ICMP is blocked on either the Stormshield or the T-Pot host, the machine won't respond. Verify whether ICMP is permitted in both places.
- **SNMP:** T-Pot does **not** install or expose an SNMP service by default. If you want to monitor the host via SNMP, you'll need to install and configure an SNMP daemon (such as `snmpd`) yourself and allow UDP/161 through both the firewall and the host firewall.

---

### 4. Verify routing

Since your management host is on a different subnet (`192.168.1.0/24`) than T-Pot (`172.16.0.0/24`), also verify:

- the Stormshield firewall permits traffic in **both directions**,
- the T-Pot host has the correct default gateway,
- return traffic is routed back through the firewall.

A simple test is:

```bash
ip route
ping 192.168.1.117
traceroute 192.168.1.117
```

---

One question that would help narrow this down:

- **Which service is actually failing?** Is it:
  - SSH on **64295**,
  - the Web UI on **64297**,
  - ICMP (ping),
  - SNMP,
  - or **all of them**?

Also, does `sudo nft list ruleset` show any rules dropping traffic from `192.168.1.0/24`?

If this solves your problem, feel free to mark it as the accepted answer so others can find it easily.
